src/collective/glossary/browser/viewlets/css_js.pt

```diff
@@ -0,0 +1,20 @@
+<tal:def define="portal_url view/portal_url">
+    <link rel="stylesheet" tal:attributes="href string:$portal_url/++resource++collective.glossary/main.css"/>
+    <link rel="stylesheet" tal:attributes="href string:$portal_url/++resource++collective.glossary/tooltip.css"/>
+
+    <script>
+        window.__define = window.define;
+        window.__require = window.require;
+        window.define = undefined;
+        window.require = undefined;
```

Comment:
why we need those lines above? I guess it's to don't mix requirejs definitions with the main Plone one?

Reply:
not needed. removed.